test/unit/logic/transactionPool.js

```diff
@@ -0,0 +1,138 @@
+  }, done, true);
+}
+
+describe('transactionPool', function () {
+  let txPool;
+
+  before(function (done) {
+    // Init transaction logic
+    async.auto({
+      rounds          : function (cb) {
+        modulesLoader.initModule(Rounds, modulesLoader.scope, cb);
+      },
+      accountLogic    : function (cb) {
+        modulesLoader.initLogicWithDb(AccountLogic, cb);
+      },
+      blockModule     : ['accountLogic', function (result, cb) {
+        modulesLoader.initModuleWithDb(BlocksModule, cb, {
+          logic: { /* dependencies not included */ },
+        });
+      }],
+      transactionLogic: ['accountLogic', function (result, cb) {
+        modulesLoader.initLogicWithDb(TransactionLogic, cb, {
+          account: result.accountLogic
+        });
+      }],
+      delegateModule  : ['transactionLogic', function (result, cb) {
+        modulesLoader.initModuleWithDb(DelegateModule, cb, {
+          logic: {
+            transaction: result.transactionLogic
+          }
+        });
+      }],
+      // transactionModule: ['transactionLogic', function (result, cb) {
+      //   modulesLoader.initModuleWithDb(TransactionModule, cb, {
+      //     transaction: result.transactionLogic
+      //   });
+      // }]
+    }, function (err, result) {
+      modulesLoader.initModuleWithDb(AccountModule, function (err, __accountModule) {
+        expect(err).to.not.exist;
+
+        var account  = __accountModule;
+        accountLogic = result.accountLogic;
+
+        // for correctly initializing setting blocks module
+        result.blockModule.lastBlock.set({ height: 10 });
+
+        result.delegateModule.onBind({
+          accounts: __accountModule,
+          // transactions: result.transactionModule,
+          blocks  : result.blockModule
+        });
+        var sendLogic = result.transactionLogic.attachAssetType(transactionTypes.SEND, new TransferLogic());
+        sendLogic.bind(account, /* rounds */ null);
+        //
+        // result.transactionModule.onBind({
+        //   accounts: __accountModule,
+        //   // transactions: result.transactionModule,
+        //   //loader:
+        // });
+
+        account.onBind({
+          delegates: result.delegateModule,
+          accounts : account,
+          // transactions: result.transactionModule
+        });
+
+        accountModuleDependencies = result;
+        txPool                    = new TransactionPool(
+          modulesLoader.scope.config.broadcasts.broadcastInterval,
+          modulesLoader.scope.config.broadcasts.releaseLimit,
+          result.transactionLogic,
+          modulesLoader.scope.bus, // bus
+          modulesLoader.logger// logger
+        );
+        txPool.bind(account, null, modulesLoader.scope.loader);
+        done();
+      }, {
+        logic: {
+          account    : result.accountLogic,
+          transaction: result.transactionLogic
+        }
+      });
+    });
+
+  });
+
+  describe('receiveTransactions', function () {
```

Comment:
hmm @karmacoma I might be blind but i don't see what's different between your formatting and mine